.gitignore

```diff
@@ -1,2 +1,10 @@
 target/
 .idea
+pom.xml.tag
```

Comment:
optional: instead of listing each of this pom.xml files, I think you can use `pom.xml.*`